inspect_byproducts.py

```diff
@@ -0,0 +1,160 @@
+"""
+<Program Name>
+  inspect_byproducts.py
+
+<Author>
+  Sachit Malik <i.sachitmalik@gmail.com>
+
+<Started>
+  June 6, 2017
+
+<Copyright>
+  See LICENSE for licensing information.
+
+<Requires>
+  in-toto
+
+<Purpose>
+  Inspections constitute an important part of in-toto. The script take the path
+  of a link file, a random string and an operator, which is used to compare
+  a certain property of the link file with the random string.
+
+  Suppose the link file is located at /user/abc/def/package.45gh325.link
```

Comment:
Ha, minor nit. This is not a hex number: `45gh325`. Could you change it to, say `45fe325`? 😆